drodrpg/DRODLib/Character.cpp

```diff
@@ -2358,6 +2361,7 @@ void CCharacter::Process(
 			break;
 			case CCharacterCommand::CC_AddRoomToMap:
 			{
+				if (bRoomBeingDisplayedOnly) return;
```

Comment:
I'm wondering whether it makes sense to "break" here to skip this command and let other safe commands run in the command sequence, instead of simply returning/halting immediately when one of these "unsafe" commands is encountered. Thoughts?

Reply:
I agree, it is much better to allow commands that would not block processing to only be skipped over rather than stopping further processing of that script. I have reviewed the commands and changed the ones I think it is best to break on instead of return.